packages/client/components/TeamPrompt/TeamPromptResponseCard.tsx

```diff
@@ -210,6 +217,35 @@ const TeamPromptResponseCard = (props: Props) => {
 
   const ref = useAnimatedCard(displayIdx, status)
 
+  const responsePermalink = makeAppURL(window.location.origin, `/meet/${meetingId}/responses`, {
+    searchParams: {
+      source: 'sharing',
```

Comment:
Including a full set of UTM params felt like overkill that users wouldn't like, but I think there's value in being able to see when a shared link was opened, hence a `source` param that's separate from the UTMs.

This is the approach Amplitude uses for its `Copy URL` functionality.

Reply:
I think shortening the full set of UTM parameters are fine but instead of `source` we should stay with `utm_source`. I did a quick experiment, it looks like GA4 wouldn't pick up the `source` parameter, but `utm_source` works:

<img width="1286" alt="Screenshot 2023-10-18 at 14 36 50" src="https://github.com/ParabolInc/parabol/assets/1879975/82f6b106-cef2-4181-a4ce-76ca57a85197">

Reply:
Fair enough! Switched this to `utm_source`.